msal/src/main/java/com/microsoft/identity/client/Account.java

```diff
@@ -129,7 +129,7 @@ public String getTenantId() {
     }
 
     @NonNull
-    String getHomeAccountId() {
+    public String getHomeAccountId() {
```

Comment:
Is this necessary? for this and `getEnvironment`

Reply:
Please describe the reasons for making such changes in the PR description

Reply:
All of the current usages are from files in the same package. We will use it from BaseStates.kt file (will be added in SiginIn PR). That file is not in the same package so we have to change the access modifier from `package` to `public`.